tamr_client/instance/__init__.py

```diff
@@ -0,0 +1,3 @@
+from tamr_client.instance import _instance, backup
```

Comment:
Don't expose non-intended interfaces:

```suggestion
from tamr_client.instance import backup
```